cg/services/pre_analysis_quality_check/sequencing_quality_checks/utils.py

```diff
@@ -0,0 +1,159 @@
+import logging
+
+from cg.constants.constants import PrepCategory
+from cg.constants.priority import Priority
+from cg.store.models import Case, Sample
+
+LOG = logging.getLogger(__name__)
+
+
+def is_sample_ready_made_library(sample: Sample) -> bool:
+    """
+    Check if a sample is a ready made library.
+
+    Returns:
+        bool: True if the sample is a ready made library, False otherwise.
+
+    """
+    passed_quality_check: bool = sample.prep_category == PrepCategory.READY_MADE_LIBRARY
+    if not passed_quality_check:
+        LOG.warning(f"Sample {sample.internal_id} is not a ready made library.")
+    return passed_quality_check
+
+
+def case_has_express_priority(case: Case) -> bool:
+    """
+    Check if a case is express priority.
+
+    Returns:
+        bool: True if the case is express priority, False otherwise.
+
+    """
+    return case.priority == Priority.express
+
+
+def sample_has_express_priority(sample: Sample) -> bool:
+    """
+    Check if a sample is express priority.
+
+    Returns:
+        bool: True if the sample is express priority, False otherwise.
+
+    """
+    return sample.priority == Priority.express
+
+
+def case_has_lower_priority_than_express(case: Case) -> bool:
+    """
+    Check if a case is lower than express priority.
+
+    Returns:
+        bool: True if the case is lower than express priority, False otherwise.
+
+    """
+    return case.priority < Priority.express
```

Comment:
```suggestion
def is_not_express_case(case: Case) -> bool:
    """
    Check if a case is lower than express priority.

    Returns:
        bool: True if the case is lower than express priority, False otherwise.

    """
    return case.priority != Priority.express
```